idoit-install

```diff
@@ -1737,6 +1737,12 @@ function prepareIDoit {
     log "Cleanup VHost directory"
     rm -rf "${INSTALL_DIR:?}/"* || abort "Unable to remove files"
     rm -f "${INSTALL_DIR}"/.htaccess || abort "Unable to remove files"
+    
+    log ""
+    log "For evaluation use \033[32mEVAL\033[0m"
+    log "If you have a license use \033[32mPRO\033[0m"
+    log "If you want the open version use \033[32mOPEN\033[0m"
+    log ""
```

Comment:
What about this?
~~~
For evaluation purposes select \033[32meval\033[0m.
If you have a valid pro license select \033[32mpro\033[0m.
Otherwise, feel free to use the free open source variant by selecting \033[32mopen\033[0m.
~~~